modules/viewing-stats-collected-kubernetes-nmtate-op.adoc

```diff
@@ -0,0 +1,90 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-stats-collected-kubernetes-nmtate-op_{context}"]
+= Viewing metrics collected by the Kubernetes NMState Operator
+
+The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and them as ready-to-use metrics.
```

Comment:
`component and them as ready-to-use metrics` <-- looks like "expose" verb is missing here?